sylius/refund-plugin/0.4/manifest.json

```diff
@@ -1,8 +1,18 @@
 {
     "bundles": {
+        "Sylius\\Bundle\\CoreBundle\\SyliusCoreBundle": ["all"],
+        "Sylius\\Bundle\\GridBundle\\SyliusGridBundle": ["all"],
+        "Sylius\\Bundle\\MailerBundle\\SyliusMailerBundle": ["all"],
+        "Sylius\\Bundle\\ResourceBundle\\SyliusResourceBundle": ["all"],
```

Comment:
These bundles is not part of the package. We should never modify other bundles in this recipe.